src/transaction/log_page_buffer.c

```diff
@@ -6946,6 +6946,19 @@ logpb_remove_archive_logs_exceed_limit (THREAD_ENTRY * thread_p, int max_count)
       if (last_arv_num_to_delete >= first_arv_num_to_delete)
 	{
 	  log_Gl.hdr.last_deleted_arv_num = last_arv_num_to_delete;
+
+	  /* Update the last_blockid needed for vacuum. Get the first page_id of the previously logged archive */
+	  new_page_id = log_Gl.append.prev_lsa.pageid;
+	  if (last_arv_num_to_delete == -1 || new_page_id == 0)
+	    {
+	      /* set as NULL_BLOCKID */
+	      logpb_update_last_blockid (thread_p, NULL_PAGEID);
```

Comment:
I don't think it is reachable. Can you please confirm?

Reply:
Indeed, with the current implementation this is no longer reachable. I am removing it.